Data/operators.csv

```diff
@@ -3,8 +3,8 @@ asObservable(),eraseToAnyPublisher(),
 asObserver(),eraseToAnySubject(),
 bind(to:),`assign(to:on:)`,Assign uses a KeyPath which is really nice and useful. RxSwift needs a Binder / ObserverType to bind to.
 buffer,buffer,
-catchError,catch,
-catchErrorJustReturn,replaceError(with:),
+catchError,catch,catchError terminates sequence with error. catch completes as usual
+catchErrorJustReturn,replaceError(with:),catchErrorJustReturn terminates sequence error. replaceError(with:) completes as usual

```

Comment:
what does completes as usual mean? Finishes with `finished` or continues as if there was no error ?

Reply:
Sorry, it's random commit. Deleted it